product-docs/installation/migrate-database.md

```
@@ -0,0 +1,122 @@
+
+#### On your host using a service
+
+Depending on your OS, install PostgreSQL — here on Ubuntu:
+
+```bash
+sudo apt install postgresql
+```
+
+Configure your PostgreSQL instance with at minimum:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)